README.md

```diff
@@ -109,4 +109,41 @@ login(userType = 'admin');        // calls login request and sets session
 - Select ui elements like a human does. Instead of `cy.get('form input.submit')`, use `cy.contains('Submit')`.
 - Do not store element as variable, like `const submitButton = cy.contains('button');`
 
-More: https://docs.cypress.io/guides/references/best-practices.html
\ No newline at end of file
+More: https://docs.cypress.io/guides/references/best-practices.html
+
+## Usage within TAO
+
+The only code that should be committed to this repo is generic code like server setup, login commands, urls, data etc.
+
+Your test code should be committed to the relevant TAO repo. Tests to do with general functionality of the platform can go in [tao-core](https://github.com/oat-sa/tao-core), while extension-specific tests might go in [taoItems](https://github.com/oat-sa/extension-tao-item), [taoGroups](https://github.com/oat-sa/extension-tao-group), etc.
+
+### Installation
+
+TODO: define this package in TAO's package.json so it can be installed by the usual `npm i`
```

Comment:
npm package was created already from this repo?

Reply:
Yes! https://www.npmjs.com/package/@oat-sa/tao-e2e-runner